cylc/flow/scripts/reinstall.py

```diff
@@ -208,6 +208,7 @@ def reinstall_cli(
         # ensure the "reinstall canceled" message shows for ctrl+c
         usr = 'n'  # cancel the reinstall
         print()    # clear the traceback line
+        return False
```

Comment:
It's too early to `return` here, this skips logic further down in the function!

Reply:
```suggestion
```